src/apps/persistence/app_listing.clj

```diff
@@ -242,17 +244,19 @@
 
 (defn- get-admin-job-stats-fields
   "Adds query fields via subselects for an app's job_count, job_count_failed, and last_used timestamp."
-  [query]
+  [query query-ops]
```

Comment:
I think you meant `query-opts` (like below) here and in `get-job-stats-fields`, but that doesn't need to be changed now.

Reply:
I'll second this one. `query-ops` makes me think, `query-operations.`